deployment/deployment.json

```diff
@@ -4742,7 +4742,7 @@
         "status": "prod",
         "versions": {
           "schema": "1.3.2",
-          "subgraph": "1.2.1",
+          "subgraph": "1.2.2",
```

Comment:
Should this be a minor change since it needs re-deployed?

Reply:
Good catch, we have been not been good ab this lately. But we can try again!